WindowsServerDocs/remote/remote-desktop-services/troubleshoot/input-method-editor-error.md

```diff
@@ -19,6 +19,9 @@ We've been made aware of the following issues with the input method editor (IME)
 Affected platforms:
 
 - Azure Virtual Desktop
-- Remote Desktop Services (so far, this issue has only been reported for Windows Server 2019)
+- Remote Desktop Services
 
-We're currently working on a solution to help solve this issue.
+Please Note:

```

Comment:
Delete this line. We don't say "please" in documentation.

Reply:
Approved